pylint/config/utils.py

```diff
@@ -27,15 +28,8 @@
 
 
 def _convert_option_to_argument(
-    opt: str, optdict: Dict[str, Any]
-) -> Union[
-    _StoreArgument,
-    _StoreTrueArgument,
-    _CallableArgument,
-    _StoreOldNamesArgument,
-    _StoreNewNamesArgument,
-    _ExtendArgument,
-]:
+    opt: str, optdict: dict[str, Any]
+) -> _StoreArgument | _StoreTrueArgument | _CallableArgument | _StoreOldNamesArgument | _StoreNewNamesArgument | _ExtendArgument:
```

Comment:
@cdce8p For some reason `black` doesn't format this. Is this a known issue? And if so, how do we want to format this ourselves?

Reply:
Not sure. You could try adding brackets and see if that helps with black.
```py
    ...
) -> (
    _StoreArgument
    | _StoreTrueArgument
    ...
)
```